launcher/src/se/llbit/chunky/launcher/JavaFxLocator.java

```diff
@@ -60,9 +60,9 @@ private static void runWithJavafx(Path javafxDir, String[] args) {
     // add the options again so the launcher can use them for chunky
     cmd.add("--javaOptions");
     StringBuilder javaOptions = new StringBuilder();
-    javaOptions.append("--module-path ");
+    javaOptions.append("--module-path \\\"");
```

Comment:
Is this intended to literally add `\"` instead of `"` to the string?

Reply:
It is required to escape twice due to how the options are given to launcher for the second invocation of the launcher that then give them to the actual chunky. Escaping once, while it does construct the correct `javaOptions`, results in the command being feed into chunky lacking the `"`.

Reply:
This would be a good comment. :+1: 
Maybe like this?

```
// escape the path twice to make the second launcher passing the options to Chunky work if it contains spaces
javaOptions.append("\\\"" + javafxDir.toAbsolutePath().toString() + "\\\"");
```

Reply:
Slightly tweaked the wording and I also adopted the string concatenation you used to clean up the line (I wasn't sure if I could do that with the stringbuilder).